service/controller/v18/cloudconfig/worker_template.go

```diff
@@ -37,6 +36,12 @@ func (c *CloudConfig) NewWorkerTemplate(ctx context.Context, customObject v1alph
 		params.Hyperkube.Kubelet.Docker.CommandExtraArgs = c.k8sKubeletExtraArgs
 		params.RegistryDomain = c.registryDomain
 		params.SSOPublicKey = c.SSOPublicKey
+
+		ignitionPath := k8scloudconfig.GetIgnitionPath(c.ignitionPath)
+		params.Files, err = k8scloudconfig.RenderFiles(ignitionPath, params)
```

Comment:
So why do we need both, Cloud Config and Ignition? Is that because of some migration path?

Reply:
That is because of test mostly. When we are running tests from the k8scloudconfig repo, the location of raw files is in package scope. When we are running tests from the operators - it is in vendoring. And in real life - it is baked into container. So `GetIgnitionPath` returns runtime path of the ignition files